### PR TITLE
Restore monitoring stack

### PR DESCRIPTION
**Pull request by aleksbykov**
- download snapshot and monitoring stack from S3
 - start monitoring stack containers
 - return link

### CODE REVIEW

File: docker/env/hydra.sh [hunk 1]
```diff
@@ -77,7 +77,7 @@ docker run --rm ${TTY_STDIN} --privileged \
     -w ${WORK_DIR} \
     -e JOB_NAME=${JOB_NAME} \
     -e BUILD_URL=${BUILD_URL} \
-    -u $(id -u ${USER}):$(id -g ${USER}) \
+    -u $(id -u ${USER}):$(grep "docker" /etc/group|cut -d: -f3) \
```
Comment (bentsi) [inline docker/env/hydra.sh]: why did you add this?

File: sdcm/utils.py [hunk 2]
```diff
@@ -239,6 +252,87 @@ def list_logs_by_test_id(test_id):
...
+    monitoring_dockers_dir = os.path.join(monitor_stack_workdir, 'scylla-grafana-monitoring-{}'.format(monitoring_branch))
+
+    for i in range(5):
+        logger.info('Start Attempt {}'.format(i + 1))
+        lr.run('cd {}; ./kill-all.sh'.format(monitoring_dockers_dir))
+        cmd = dedent("""cd {monitoring_stack_dir};
+                ./start-all.sh \
+                -s {monitoring_stack_dir}/config/scylla_servers.yml \
+                -n {monitoring_stack_dir}/config/node_exporter_servers.yml \
+                -d {monitoring_data_dir} -v {scylla_version}""".format(monitoring_stack_dir=monitoring_dockers_dir,
```
Comment (bentsi) [inline sdcm/utils.py]: use the same variable in formatted string and format with **locals()

File: sdcm/utils.py [hunk 3]
```diff
@@ -239,6 +252,87 @@ def list_logs_by_test_id(test_id):
...
+        if not result.ok:
+                logger.warning("During restoring file {} next errors occured:\n {}".format(arch['link'], result))
+                return False
+        logger.info("Extract data finished")
+
+    logger.info('Monitoring stack files available {}'.format(monitor_stack_workdir))
+
+    monitoring_dockers_dir = os.path.join(monitor_stack_workdir, 'scylla-grafana-monitoring-{}'.format(monitoring_branch))
+
+    for i in range(5):
```
Comment (bentsi) [inline sdcm/utils.py]: separate in different function and using retrying decorator

File: sdcm/utils.py [hunk 4]
```diff
@@ -239,6 +252,87 @@ def list_logs_by_test_id(test_id):
...
+        res = lr.run(cmd, ignore_status=True)
+        if res.ok:
+            r = lr.run('docker ps')
+            logger.info(r.stdout.encode('utf-8'))
+            break
+        else:
+            logger.warning(res)
+            logger.info('Start failed, Retry ...')
+    else:
+        logger.warning('Docker containers for Monitoring stack was not started. See docker container logs for details')
```
Comment (bentsi) [inline sdcm/utils.py]: *were* not started

File: sdcm/utils.py (hunk 2 above)
Comment (aleksbykov) [reply sdcm/utils.py]: fixed

File: sdcm/utils.py (hunk 4 above)
Comment (aleksbykov) [reply sdcm/utils.py]: fixed

File: sdcm/utils.py (hunk 3 above)
Comment (aleksbykov) [reply sdcm/utils.py]: done
